pkgs/development/compilers/sbcl/common.nix

```diff
@@ -22,6 +22,12 @@ stdenv.mkDerivation rec {
 
   buildInputs = [texinfo];
 
+  patches = lib.optional (lib.versionAtLeast version "2.1.2") (fetchpatch {
```

Comment:
Shouldn't there also be an upper bound for the version if the fix is already committed?

Reply:
The fix is already committed but not yet released as far as I can tell, so I don't know what version should be the upper bound - I could assume that the current version (2.1.7) is the last one which won't contain the commit, but who knows. I figured that it would be easy enough to add an upper bound when Nixpkgs starts including a version to which the patch doesn't apply anymore, since the build will fail.

Reply:
Given SBCL branch list, seems to be safe to assume whatever is in master will be included in 2.1.8…

Reply:
OK, I added 2.1.8 as an upper bound. Thanks!